flow/templates/base_status.jinja

```diff
@@ -26,14 +26,17 @@
 {% endfor %}
 {% endblock%}
 
-{% block operation_summary %}
-| operation | number of eligible jobs |
-| --------- | ----------------------- |
-{% for op, n_jobs in op_counter %}
-| {{ op }} | {{ n_jobs }} |
+{% block aggregate_summary %}
+| operation | number of eligible aggregates |
```

Comment:
I might just use the word operations here. I think it is less confusing and all operations are aggregates so it isn't false.